PocketCoffea/parameters/triggers.py

```diff
@@ -65,5 +65,38 @@
                 "HLT_IsoMu24",
             ],
         },
+    },
+    "semileptonic" : {
+        "2016" : {
+            "e" : [
+            ],
+            "mu" : [
+            ]
+        },
+        "2017" : {
+            "e" : [
+            ],
+            "e_flag" : [
+            ],
+            "mu" : [
+            ]
+        },
+        "2018" : {
+            "e" : [
+                "Ele32_WPTight_Gsf",
+                "Ele32_WPTight_Gsf_L1DoubleEG",
+                "Ele28_eta2p1_WPTight_Gsf_HT150"
+            ],
+            "mu" : [
+                "IsoMu24_eta2p1_TightChargedIsoPFTauHPS35_Trk1_eta2p1_Reg_CrossL1",
+                "IsoMu24_eta2p1_MediumChargedIsoPFTauHPS35_Trk1_TightID_eta2p1_Reg_CrossL1",
+                "IsoMu24_eta2p1_TightChargedIsoPFTauHPS35_Trk1_TightID_eta2p1_Reg_CrossL1",
+                "IsoMu24_eta2p1_MediumChargedIsoPFTauHPS35_Trk1_eta2p1_Reg_CrossL1",
+                "IsoMu24",
+                "IsoMu24_eta2p1",
+                "IsoMu24_TwoProngs35",
```

Comment:
I'm not sure including everything is the best choice. For example the TwoProngs triggers I don't know what it does.

Reply:
Sure, we will have to check which triggers need to be applied.